src/services/user/services/userService.js

```diff
@@ -3,6 +3,7 @@ const { authenticate } = require('@feathersjs/authentication');
 const { iff, isProvider, disallow } = require('feathers-hooks-common');
 // const logger = require('../../../logger');
 const { modelServices: { prepareInternalParams } } = require('../../../utils');
+const { createChannel } = require('../../../utils/rabbitmq');
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['createChannel' is assigned a value but never used.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5276619)